### PR TITLE
fix, test extracting .conda given as relative path

### PR DESCRIPTION
**Pull request by dholth**
Test for #115

Can extract `/full/path/to/*.conda` but not `*.conda`

### CODE REVIEW

**Comment by dholth** [comment]
Fix #115

File: src/conda_package_handling/api.py [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
+
 # expose these two exceptions as part of the API.  Everything else should feed into these.
 from .exceptions import ConversionError, InvalidArchiveError  # NOQA
 from .tarball import CondaTarBZ2 as _CondaTarBZ2, libarchive_enabled  # NOQA
 from .conda_fmt import CondaFormat_v2 as _CondaFormat_v2
 from .utils import (TemporaryDirectory as _TemporaryDirectory, rm_rf as _rm_rf,
                     get_executor as _get_executor)
 
-SUPPORTED_EXTENSIONS = {'.tar.bz2': _CondaTarBZ2,
+SUPPORTED_EXTENSIONS: Dict[str, AbstractBaseFormat] = {'.tar.bz2': _CondaTarBZ2,
```
Comment (dholth) [inline src/conda_package_handling/api.py:18]: pylance doesn't think _CondaTarBZ2 is a AbstractBaseFormat?

File: src/conda_package_handling/conda_fmt.py [hunk 2]
```diff
@@ -23,18 +23,17 @@ def _lookup_component_filename(zf, file_id, component_name):
     return component_filename
 
 
-def _extract_component(fn, file_id, component_name, dest_dir=os.getcwd()):
```
Comment (dholth) [inline src/conda_package_handling/conda_fmt.py:26]: called from one place, never with an assumed dest_dir; `os.getcwd()` called once when imported isn't a good idea

File: src/conda_package_handling/interface.py [hunk 3]
```diff
@@ -4,6 +4,11 @@
 
 class AbstractBaseFormat(metaclass=abc.ABCMeta):
 
+    @staticmethod
+    @abc.abstractmethod
+    def supported(fn):
+        return False
+
```
Comment (kenodegard) [inline src/conda_package_handling/interface.py:11]: OOP at its finest

**Review by kenodegard — APPROVED** [review]

**Pull request merged**